Code/Tools/ProjectManager/Source/UpdateProjectCtrl.cpp

```diff
@@ -117,6 +128,16 @@ namespace O3DE::ProjectManager
             m_stack->setCurrentWidget(m_gemRepoScreen);
             Update();
         }
+        else if (screen == ProjectManagerScreen::GemCatalog)
+        {
+            m_stack->setCurrentWidget(m_gemCatalogScreen);
```

Comment:
huh, if QStackWidget has a signal for when the current widget changes we could just call Update every time that fires to avoid having to paste Update() all over.